### PR TITLE
Possibility to visualize saved flight plans

### PR DESCRIPTION
**Pull request by kornrumpfl**
-by clicking on a saved flight plan visualize this flight plan
-spelling correction
-disable past dates

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -78,6 +79,16 @@ function App() {
     );
   }
 
+  function loadFlightPlan(flightPlanId) {
+    console.log(flightPlanId);
```
Comment (Kheirah) [inline src/App.js:83]: you may remove unused logs

File: src/components/Main.js [hunk 2]
```diff
@@ -109,7 +119,12 @@ export default function Main({ onHandleSubmit }) {
       <Time>
         <h2>Flight Departure Date/Time</h2>
         <div>
-          <input id="flightDate" type="date" aria-label="date" />
+          <input
+            id="flightDate"
+            type="date"
+            aria-label="date"
+            min={getDate()}
```
Comment (Kheirah) [inline src/components/Main.js:126]: I can still input dates prior to today and it's accepted by the input field. In other words, it's not working 🤣

File: src/components/flightPlan/flightPlan.js [hunk 3]
```diff
@@ -13,15 +13,14 @@ export default function FlightPlan({ flightPlanData, onSavePlan }) {
   const position = [53.633354, 9.999303];
   const navigate = useNavigate();
   console.log(flightPlanData);
```
Comment (Kheirah) [inline src/components/flightPlan/flightPlan.js:15]: remove logs

**Review by Kheirah — CHANGES_REQUESTED** [review]
Overall, great progress!

The one major hiccup is the check on past dates. Please look into it and find a fix :)

File: src/components/Main.js (hunk 2 above)
Comment (kornrumpfl) [reply src/components/Main.js:126]: i notice it a few minutes ago!

**Pull request merged**